### PR TITLE
Improve and align documentation

### PR DESCRIPTION
**Pull request by lukas-bednar**

### CODE REVIEW

File: roles/ovirt-engine-backup/README.md [hunk 1]
```diff
@@ -14,28 +20,31 @@ Pre-installed and running ovirt.
...
-      remote_user: root
-    vars:
-        ovirt_backup_mode: 'backup'
-        ovirt_backup_archive: '/tmp/engine-backup.gzip'
-        ovirt_backup_log_file: '/tmp/engine-backup.log'
-        ovirt_backup_scope: 'all'
-    roles:
-      - { role: ovirt-engine-bakcup }
+```yaml
+- hosts: engine
```
Comment (lukas-bednar) [inline roles/ovirt-engine-backup/README.md]: add ```---```

File: roles/ovirt-engine-backup/README.md [hunk 2]
```diff
@@ -14,28 +20,31 @@ Pre-installed and running ovirt.
 Role Variables
 --------------
 
-    ovirt_backup_mode: backup'restore
-    ovirt_backup_archive: Path to archive file
-    ovirt_backup_log_file: Log file, downloaded in case of error
-    ovirt_backup_scope: all|files|db|dwhdb|reportsdb
-    
+```yaml
+ovirt_backup_mode: backup'restore
```
Comment (lukas-bednar) [inline roles/ovirt-engine-backup/README.md]: add ```---```

File: roles/ovirt-engine-dwh-remote/README.md [hunk 3]
```diff
@@ -1,20 +0,0 @@
-Role Name
```
Comment (lukas-bednar) [inline roles/ovirt-engine-dwh-remote/README.md]: This role is not implemented, I don't see reason to keep it in repo

File: roles/ovirt-engine-install-packages/README.md [hunk 4]
```diff
@@ -1,45 +1,37 @@
...
-        # role-vars: ovirt-engine-install-packages
-        ovirt_engine_type: 'ovirt-engine'
-        ovirt_engine_dwh: True
-        ovirt_engine_version: '3.6'
-
-      roles:
-        - { role: common }
-        - { role: engine-install }
+```yaml
+- hosts: engine
```
Comment (lukas-bednar) [inline roles/ovirt-engine-install-packages/README.md]: add ```---```

File: roles/ovirt-engine-remote-db/README.md [hunk 5]
```diff
@@ -39,26 +46,26 @@ None
...
-        ovirt_engine_db_user: 'engine'
-        ovirt_engine_db_password: 123456
-        ovirt_engine_dwh_db_name: 'ovirt_engine_history'
-        ovirt_engine_dwh_db_user: 'ovirt_engine_history'
-        ovirt_engine_dwh_db_password: 123456
-
-    roles:
-      - { role: ovirt-engine-remote-db }
+```yaml
+- hosts: database
```
Comment (lukas-bednar) [inline roles/ovirt-engine-remote-db/README.md]: add ```---```

File: roles/ovirt-engine-setup/README.md [hunk 6]
```diff
@@ -11,66 +16,49 @@ ovirt_engine_answer_file_path.
...
-    ovirt_engine_db_user: DB user which can access ovirt-engine DB (default: 'engine')
-    ovirt_engine_db_password: password for user of ovirt-engine DB
-    ovirt_engine_dwh_db_host: IP or hostname of PostgreSQL server for DWH database (default: 'localhost')
-    ovirt_engine_dwh_db_port: Server listening port (default 5432)
-    ovirt_engine_dwh_db_name: DB name for ovirt-engine-dwh (default: 'ovirt_engine_history')
-    ovirt_engine_dwh_db_user: DB user which can access ovirt-engine-dwh DB (default: 'ovirt_engine_history')
-    ovirt_engine_dwh_db_password: password for user of ovirt-engine DB
-
+```yaml
+ovirt_engine_type: Type of product 'ovirt-engine' - for installing oVirt product
```
Comment (lukas-bednar) [inline roles/ovirt-engine-setup/README.md]: add ```---```

File: roles/ovirt-engine-setup/README.md [hunk 7]
```diff
@@ -11,66 +16,49 @@ ovirt_engine_answer_file_path.
...
-        ovirt_engine_dwh_db_port: 5432
-        ovirt_engine_dwh_db_name: 'ovirt_engine_history'
-        ovirt_engine_dwh_db_user: 'ovirt_engine_history'
-        ovirt_engine_dwh_db_password: '37xmBKECANQGm0z3SfylMp'
-
-      roles:
-        - { role: common }
-        - { role: engine-install }
+```yaml
+- hosts: engine
```
Comment (lukas-bednar) [inline roles/ovirt-engine-setup/README.md]: add ```---```

**Comment by lukas-bednar** [comment]
@StLuke @pkubica @vkondula could you please review it ?
read that changes carefully, whether doc & examples reflect reality.

File: roles/ovirt-engine-config/README.md [hunk 8]
```diff
@@ -42,14 +48,14 @@ Example Playbook
...
-      value: false
-      version: "general"
+    ovirt_engine_config:
+      -
+        key: "EnableMACAntiSpoofingFilterRules"
+        value: false
+        version: "general"
   roles:
     -
       role: "ovirt_engine_config"
```
Comment (lukas-bednar) [inline roles/ovirt-engine-config/README.md]: ovirt-engine-config

File: roles/ovirt-engine-remote-db/README.md [hunk 9]
```diff
@@ -1,13 +1,17 @@
...
+oVirt Engine Remote DB
+======================
 
-Remote DB setup
-- install and configure PostgreSQL server on machine (latest from repository) 
+- install and configure PostgreSQL server on machine (latest from repository)
 - create necessary roles and databases for engine and DWH (based on variables)
 - update firewalld and iptables (based on service present)
 
 - If PostgreSQL server is already installed then create only roles and databases
```
Comment (vkondula) [inline roles/ovirt-engine-remote-db/README.md]: use all lower case or all uppercase at the beginning of sentence

File: roles/ovirt-engine-backup/README.md (hunk 2 above)
Comment (lukas-bednar) [inline roles/ovirt-engine-backup/README.md]: Done

File: roles/ovirt-engine-backup/README.md (hunk 1 above)
Comment (lukas-bednar) [inline roles/ovirt-engine-backup/README.md]: Done

File: roles/ovirt-engine-remote-db/README.md (hunk 5 above)
Comment (lukas-bednar) [inline roles/ovirt-engine-remote-db/README.md]: Done

File: roles/ovirt-engine-setup/README.md (hunk 6 above)
Comment (lukas-bednar) [inline roles/ovirt-engine-setup/README.md]: Done

File: roles/ovirt-engine-setup/README.md (hunk 7 above)
Comment (lukas-bednar) [inline roles/ovirt-engine-setup/README.md]: Done

File: README.md [hunk 10]
```diff
@@ -1,46 +1,91 @@
...
 
-second, fill Install_engine.yml file
-README for this file can be found in role engine-install readme
+### hypervisors
 
+This host groups is meant for all nodes which are supposed to be used as
+hypervisors of oVirt Engine.
 
-command for run playbook:
+Note that none of roles in this repository adds these machines into
```
Comment (nellyc) [inline README.md]: none of the roles in this...

File: README.md [hunk 11]
```diff
@@ -1,46 +1,91 @@
...
+Ansible roles for oVirt deployment.
 
+Roles in this repository can configure nodes used as oVirt deployment members.
 
-## Requirements
+## Host groups
 
-Package sshpass installed for authentication
+Each role has documentation, where you can find ``Target systems`` section.
+This section says on what kind of node you can apply relevant role.
```
Comment (nellyc) [inline README.md]: This section specifies on what kind of node...

File: README.md (hunk 10 above)
Comment (nellyc) [inline README.md]: note that none of the roles in this...

File: README.md [hunk 12]
```diff
@@ -1,46 +1,91 @@
...
-        [engine]
-        IP or FQDN
+This host group is meant for node where the oVirt Engine management is supposed
+to be deployed.
 
-second, fill Install_engine.yml file
-README for this file can be found in role engine-install readme
+### hypervisors
 
+This host groups is meant for all nodes which are supposed to be used as
```
Comment (nellyc) [inline README.md]: host groups are meant

File: README.md [hunk 13]
```diff
@@ -1,46 +1,91 @@
...
+hypervisors of oVirt Engine.
 
-command for run playbook:
+Note that none of roles in this repository adds these machines into
+oVirt Engine.
 
-        ansible-playbook -i example/inventory/install_engine.inv example/playbooks/install_engine.yml
+### database
+
+This host group is meant for node where database is supposed to be deployed.
```
Comment (nellyc) [inline README.md]: for node on which the database...

File: README.md [hunk 14]
```diff
@@ -1,46 +1,91 @@
...
+
+### dwh
+
+This host group is meant for node where DWH is supposed to be deployed.
```
Comment (nellyc) [inline README.md]: same

File: README.md [hunk 15]
```diff
@@ -1,46 +1,91 @@
...
+    - role: ovirt-common
+```
+
+Minimal inventory example
+
+```ini
+[all:vars]
+ovirt_engine_type=ovirt-engine
+ovirt_engine_version=4.1
+# Make sure that link to release rpm is working!!!
```
Comment (nellyc) [inline README.md]: im not sure we should use release rpm term here - its internal working method

File: roles/ovirt-common/README.md [hunk 16]
```diff
@@ -1,24 +1,28 @@
-Role Name
-=========
+oVirt Common
+============
 
-Repository setup
-Role install necessary repositories
+This role installs relevant repositories on target system.
+It role is meant to be used on all systems which are members of
```
Comment (nellyc) [inline roles/ovirt-common/README.md]: this role is mean

File: README.md (hunk 15 above)
Comment (nellyc) [inline README.md]: i take it back :)
it is published upstream, so i guess its ok

**Pull request merged**